src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1331,6 +1337,11 @@ private bool DisplayFutureFileMessage(string fullFilePath, Version fileVersion,
             return args.ClickedButtonId == (int)ButtonId.Proceed;
         }
 
+        private void OnWorkspacePropertyChanged(object sender, PropertyChangedEventArgs args)
+        {
+            if (args.PropertyName == "RunEnabled")
+                OnPropertyChanged("RunEnabled");
+        }
```

Comment:
During the discussion, I was tempting to make this `OnPropertyChanged(args.PropertyName)` instead of just `RunEnabled`, but I think this is good for now. We might not want all the `WorkspaceModel` properties get notified as `DynamoModel` properties.